readme.md

```diff
@@ -72,7 +72,7 @@ String where the Unicode symbols will be replaced with fallback symbols dependin
 | tick                                        | `✔`  |   `√`   |
 | info                                        | `ℹ`  |   `i`   |
 | warning                                     | `⚠`  |   `‼`   |
-| cross                                       | `✖`  |   `×`   |
+| cross                                       | `✘`  |   `×`   |
```

Comment:
The following test needs to be fixed:

https://github.com/sindresorhus/figures/blob/62b5c69bd1ff54a523857b89f1fad184fb553ba5/test.js#L17-L18

Reply:
Thanks, updated.